conda_forge_tick/update_upstream_versions.py

```diff
@@ -482,7 +579,7 @@ def update_upstream_versions(
     gx: nx.DiGraph, sources: Iterable[AbstractSource] = None,
 ) -> None:
     sources = (
-        (PyPI(), Github(), NPM(), CRAN(), ROSDistro(), RawURL())
+        (PyPI(), CRAN(), Github(), NPM(), ROSDistro(), RawURL())
```

Comment:
Guess this is to ensure we do well behaved ones first

Reply:
Why was https://github.com/regro/cf-scripts/pull/166/files reverted?

Reply:
Well some of the changes in this PR made rawurl take a lot longer. Thus I reordered then. 

Why did we put rawurl first anyways?

Reply:
I switched this back. We should all, me included, be better about documenting things like this in the future. This will avoid regressions when bugs get unfixed.